skiko/gradle.properties

```diff
@@ -1,5 +1,6 @@
 kotlin.code.style=official
 deploy.version=0.0.0
+org.jetbrains.kotlin.native.version=1.5.20-dev-5723
```

Comment:
`Project property 'org.jetbrains.kotlin.native.version' is deprecated. Please use 'kotlin.native.version' instead.`

Reply:
chenged